src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -284,7 +359,7 @@ public async Task HeadersFrame_IdleStream_ConnectionError()
                 Task sendTask = client.GetAsync(server.Address);
 
                 await server.EstablishConnectionAsync();
-                await server.ReadRequestHeaderAsync();
+                int streamId = await server.ReadRequestHeaderAsync();
```

Comment:
Is there any reason to add this line if we don't use the streamId?

Reply:
Not really, probably a copy-and-paste thing.